docs(web-modeler): add environment variables section to git-sync (#5008)
camunda/camunda-docs: docs/self-managed/modeler/web-modeler/configuration/configuration.md

Refer to the [advanced SSL configuration guide](./ssl.md) for additional details on how to set up secure connections (incoming & outgoing) to the Web Modeler components.

### Git Sync

> Web Modeler supports syncing files via [Git Sync](../../../../components/modeler/web-modeler/git-sync.md). Providing the base url for you provider is needed if you have self-hosted GitLab or GitHub instances.

Reviewer: I'd suggest a small rephrasing:
```suggestion
Web Modeler supports syncing files via [Git Sync](../../../../components/modeler/web-modeler/git-sync.md). You need to provide the base url for you provider if you are using a self-hosted GitLab or GitHub instance.
```

> | Provider      | Variable                                  | Description                                                                                                                   | Default                     |

Reviewer: Just to be aligned with the other tables on this page:
```suggestion
| Provider      | Environment variable                      | Description                                                                                                                   | Default value               |
```
